packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -90,6 +90,12 @@ Use a drilldown menu when data is structured into levels and includes a long lis
 
 If data is more complex and has more than 2 levels, use a drilldown menu with breadcrumbs instead. Breadcrumbs offer better navigation between different levels. The number of visited levels is shown in a grey badge with a dropdown menu showing the menu items between the first and last level. The badge allows you to go back to a specific visited level. If you need to see the whole structure of your data, use a [tree view](/components/tree-view) instead. 
 
+### Red text menu
```

Comment:
Added red text menu section

Reply:
I would edit this section a little bit to have a bit more description, and to not just be bullet points.

Suggestion:
If you have destructive items in a dropdown menu, you can optionally use red text styling for that item. A divider should be used to separate the destructive menu items from the non-destructive ones.